lib/protobuf/field/varint_field.rb

```diff
@@ -32,6 +42,11 @@ def self.encode(value)
         (bytes << value).pack('C*')
       end
 
+      # Load the cache of VarInts on load of file
+      (0..CACHE_LIMIT).to_a.each do |cached_value|
```

Comment:
@zachmargolis we can include the update in the next release (I will be reviewing the PR's submitted today, tonight)

it only runs during loading of the file, so shouldn't have a running impact